git_repo/services/ext/gitlab.py

```diff
@@ -16,8 +16,8 @@ class GitlabService(RepositoryService):
     fqdn = 'gitlab.com'
 
     def __init__(self, *args, **kwarg):
-        self.gl = gitlab.Gitlab(self.url_ro)
-        super(GitlabService, self).__init__(*args, **kwarg)
+        super().__init__(*args, **kwarg)
+        self.gl = gitlab.Gitlab(self.url_ro, ssl_verify=not self.insecure)
```

Comment:
I'm not sure if the underlying libraries warn. I need to test again. It's probably a good idea to yell abaut insecure connections, but with the explicit 'insecure' option it should be fairly clear.